string-manipulation-1/src/reverse-word.js

```diff
@@ -0,0 +1,15 @@
+/* exported reverseWord */
+/**
+ create a function that returns a string of a word in reverse order
+ create a storage container
+ pull each letter of the string into the storage container but start from the end of the word
+once finished, return the new string
+ */
+
+function reverseWord(word) {
+  var newString = '';
+  for (var i = 1; i <= word.length; i++) {
+    newString = newString.concat(word[word.length - i]);
```

Comment:
`concat` is a method intended for arrays, you should just use the `+` operator

Reply:
okay cool, will use += from now on !